crime-means-assessment/src/test/java/uk/gov/justice/laa/crime/meansassessment/data/builder/TestModelDataBuilder.java

```diff
@@ -346,7 +342,7 @@ public static ApiAssessmentSectionSummary getApiAssessmentSectionSummary() {
                                         new ApiAssessmentDetail()
                                                 .withCriteriaDetailId(132)
                                                 .withApplicantAmount(TEST_APPLICANT_VALUE)
-                                                .withApplicantFrequency(TEST_FREQUENCY)
+                                                .withApplicantFrequency(Optional.of(TEST_FREQUENCY))
```

Comment:
I would make use of static import in all places where Optional is added
`.withApplicantFrequency(of(TEST_FREQUENCY))`.

Reply:
I would be okay either way. Possibly readability is better without the static import :)

Reply:
IMO the less the verbosity the better the code readability. But it all goes down to personal preferences.